src/test/scala/edu/ie3/simona/model/grid/RefSystemSpec.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.model.grid
+
+import edu.ie3.util.scala.quantities.Sq
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import squants.{Dimensionless, Each}
+import squants.electro.{Amperes, ElectricPotential, Kilovolts, Ohms, Volts}
+import squants.energy.{Megawatts, Power, Watts}
+
+class RefSystemSpec extends AnyFlatSpec with Matchers {
+
+  "A RefSystem with nominal power and nominal voltage" should "provide corresponding nominal current and nominal impedance" in {
+
+    val nominalPowerAsSquant: Power = Sq.create(600000, Watts)
+    val nominalVoltageAsSquant: ElectricPotential = Sq.create(10000, Volts)
```

Comment:
```suggestion
    val nominalPowerAsSquant: Power = Kilowatts(600)
    val nominalVoltageAsSquant: ElectricPotential = Kilovolts(10)
```
almost there :)

Reply:
Same for the other squants you created by Sq.create(). The Sq.create was/is a workaround we need within the groovy tests. For scala you can directly use the unit.

Reply:
Now I understand it, thanks!